src/app.jsx

```diff
@@ -391,6 +396,9 @@ const DropdownWithKebab = ({ selected, path, setPath, setPathIndex }) => {
                 </MenuToggle>}
         >
             <DropdownList>
+                <DropdownItem id="create-item" itemId='create-item' key="create-item" onClick={createDirectory}>
```

Comment:
itemId does  not seem to be a documented property for dropdown  item. https://patternfly-react-v5.surge.sh/components/menus/dropdown/#dropdownitem

Can you remove this from this and the delete entry just bellow?